implementations/rust/ockam/ockam_command/tests/bats/unit.bats

```diff
@@ -358,6 +358,38 @@ teardown() {
   assert_output --partial "NotFound"
 }
 
+@test "portals - delete a tcp inlet" {
```

Comment:
We should rename this to `portals - CRUD`.

Reply:
Should be solved in latest commit